### ISSUE TITLE
@JsonIgnore on getter and @JsonProperty on setter 

### ISSUE THREAD

**Issue by bcliao** [stoicflame/enunciate#834]
Hi, 
I have a property in a POJO which allow to be set by client and always return null (e.g. password) so I add `@JsonIgnore` on getter, however by this way the property is not generated in both doc and client code (java-json-client). 
According to https://github.com/FasterXML/jackson-annotations/wiki/Jackson-Annotations, 

> `@JsonIgnore`: simple annotation to use for ignoring specified properties:
Only needs to be added to one of accessors/mutators (field, getter/setter, constructor parameter), but will have effect on the "whole" property: that is, adding annotation to a "getter" will also disable "setter"
... unless "setter" has `@JsonProperty`, in which case this is considered a "split property" with enabled "setter" but no "getter" ("read-only", so that property may be read from input, but is not written output)

I add `@JsonProperty` on setter but still does not work. Any idea?

**Comment by stoicflame** [comment]
Enunciate currently doesn't support "split properties".

How would you expect a split property to show up in the documentation? Just as if it were a normal property?

**Comment by bcliao** [comment]
Yes, I think it is good enough for this case. Developer, like me, should add reasonable document to remind the API user the property is read only or write only. What do you think?

**Comment by stoicflame** [comment]
[Enunciate 2.11 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.11.0).
